src/Microsoft.ApplicationInsights.AspNet/SdkVersionUtils.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.ApplicationInsights.AspNet
+{ 
+    using System.Linq;
+    using System.Reflection;
+
+    internal class SdkVersionUtils
+    {
+        internal const string VersionPrefix = "aspnet5: ";
+
+        internal static string GetAssemblyVersion()
+        {
+            return typeof(SdkVersionUtils).GetTypeInfo().Assembly.GetCustomAttributes<AssemblyFileVersionAttribute>()
+                    .First()
```

Comment:
(Reflection + Linq) x 2 middlewares per request. :scream: Can we cache this value?